pkg/term/term_linux_test.go

```diff
@@ -3,29 +3,20 @@
 package term
 
 import (
-	"flag"
 	"io/ioutil"
 	"os"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-var rootEnabled bool
-
-func init() {
-	flag.BoolVar(&rootEnabled, "test.root", false, "enable tests that require root")
```

Comment:
Could it be related to running the tests bare-metal? I seem to recall there were some tests in the past that could screw up the host (we had a fun one changing system time to verify certificate expiry)

ping @tiborvass @andrewhsu could you have a quick look to verify we can remove this?